package.json

```diff
@@ -18,10 +18,12 @@
   "homepage": "https://github.com/bugsnag/bugsnag-cli#readme",
   "dependencies": {
     "axios": "^1.4.0",
-    "console.table": "^0.10.0"
+    "console.table": "^0.10.0",
+    "js-yaml": "^4.1.0"
   },
   "files": [
-    "install.js"
+    "install.js",
+    "yml"
```

Comment:
Does this need to be `supported-platforms.yml` ?

Reply:
Ah, it should be. I thought including just yml would do all files with that extension. It didn't! But after making this change it works:
```
 > tar -tvf bugsnag-cli-2.0.0.tgz                                                                                                                                     
-rw-r--r--  0 0      0        1051 Oct 26  1985 package/LICENSE
-rw-r--r--  0 0      0        2557 Oct 26  1985 package/install.js
-rw-r--r--  0 0      0         667 Oct 26  1985 package/package.json
-rw-r--r--  0 0      0        5564 Oct 26  1985 package/README.md
-rw-r--r--  0 0      0         695 Oct 26  1985 package/supported-platforms.yml
```